packages/monaco/src/browser/style/index.css

```diff
@@ -41,7 +41,7 @@
 /* List highlight, see https://github.com/microsoft/vscode/blob/ff5f581425da6230b6f9216ecf19abf6c9d285a6/src/vs/workbench/browser/style.ts#L50 */
 .monaco-tree .monaco-tree-row .monaco-highlighted-label .highlight,
 .monaco-list .monaco-list-row .monaco-highlighted-label .highlight {
-  color: var(--theia-list-highlightForeground) !important;
+  color: var(--theia-list-highlightForeground);
```

Comment:
Unfortunately additional styling is likely necessary as this breaks the highlighting when using quick-inputs (blue fuzzy matching should be red in this case):

![image](https://user-images.githubusercontent.com/40359487/225622770-5daa590e-9d85-4eaa-9cf9-2ae68fc4bdde.png)

Reply:
hmm, I'm missing this case..how about add `!important` to `focusHighlightForeground` in suggest widget?

```css
// which was in @theia/monaco-editor-core/esm/vs/editor/contrib/suggest/browser/media/suggest.css

.monaco-editor .suggest-widget .monaco-list .monaco-list-row.focused>.contents>.main .monaco-highlighted-label .highlight {
	color: var(--vscode-editorSuggestWidget-focusHighlightForeground) !important;
}
```

i have test suggest & quick input feature in light/dark/red theme, still work for me.


https://user-images.githubusercontent.com/3282495/225678125-b2ee1438-a6ea-4170-926d-d3a3c7189ebc.mp4

Reply:
The use-case in case it wasn't clear was to use <kbd>F1</kbd> (quick-command palette) and perform a search.
The final solution should fix the suggest widget as you've done but not break quick-input menus.

Reply:
yep, but how can i contribute this solution to @theia/monaco-editor-core?

Reply:
> yep, thank for your remind, but how to contribute this solution to `@theia/monaco-editor-core`?

I don't believe changes in `@theia/monaco-editor-core` is necessary, we should be able to fix the behavior in our sources. The `@theia/monaco-editor-core` package is built from VS Code's sources which allows us to consume the monaco-editor and other related functionality.